pageserver/src/tenant/timeline.rs

```diff
@@ -242,6 +242,13 @@ pub struct Timeline {
     pub delete_lock: tokio::sync::Mutex<bool>,
 
     eviction_task_timeline_state: tokio::sync::Mutex<EvictionTaskTimelineState>,
+
+    /// Barrier to wait before doing initial logical size calculation. Used only during startup.
+    initial_logical_size_can_start: Option<completion::Barrier>,
+
+    /// Completion shared between all timelines created during startup; used to delay heavier
+    /// background tasks until some logical sizes have been calculated.
+    initial_logical_size_attempt: Mutex<Option<completion::Completion>>,
```

Comment:
```suggestion
    /// Completion shared between all timelines activated during startup; used to delay heavier
    /// background tasks until some logical sizes have been calculated.
    initial_logical_size_attempt: Mutex<Option<completion::Completion>>,
```

Reply:
This is passed through `Timeline::new` so I was thinking of created, but it is true that creating during startup implies activating.

Reply:
My point is that "timeline create", in my book, means that we create a new timeline on disk.

What you meant by "create" was the allocation of the timeline object in memory during pageserver startup.
IMO that's a confusing use of "create", hence I opted for activate.

We can also use `loaded`, I guess that's actually better.